server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -5,20 +5,30 @@
     function TimelineCtrl($interval, PagesManager, blogsService, config, $anchorScroll, $timeout, Permalink, transformBlog) {
 
         var POSTS_PER_PAGE = config.settings.postsPerPage;
+        var STICKY_POSTS_PER_PAGE = 100;
         var PERMALINK_DELIMITER = config.settings.permalinkDelimiter || '?';
         var DEFAULT_ORDER = config.settings.postOrder; // newest_first, oldest_first or editorial
         var UPDATE_MANUALLY = config.settings.loadNewPostsManually;
         var UPDATE_EVERY = 10*1000; // retrieve update interval in millisecond
         var vm = this;
-        var pagesManager = new PagesManager(POSTS_PER_PAGE, DEFAULT_ORDER),
+        var pagesManager = new PagesManager(POSTS_PER_PAGE, DEFAULT_ORDER, false),
```

Comment:
these PageManager constructors (in theme and editor) start to be hard to be understandable.
Shouldn't we use a single object as parameter, then we can name the properties ?